core/src/main/scala/org/apache/spark/deploy/master/WorkerInfo.scala

```diff
@@ -107,4 +107,8 @@ private[spark] class WorkerInfo(
   def setState(state: WorkerState.Value): Unit = {
     this.state = state
   }
+
+  def isAlive(): Boolean={
```

Comment:
No parens, return type or braces are needed. In fact is it worth writing a method?